sdk/batch/Microsoft.Azure.Batch.Conventions.Files/src/Microsoft.Azure.Batch.Conventions.Files.csproj

```diff
@@ -12,15 +12,14 @@
     <PackageReleaseNotes>
       Improve usability of GetOutputStoragePath.
     </PackageReleaseNotes>
-    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <SignAssembly>true</SignAssembly>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="System.Text.Encodings.Web" />
-    <PackageReference Include="Microsoft.Azure.Batch" />
+    <PackageReference Include="Microsoft.Azure.Batch" Version="15.1.0" />
```

Comment:
This shouldn't be needed - it should pick up the version from the Packages.Data.Props.

This actually shows up as an error in Visual Studio:

```
Severity	Code	Description	Project	File	Line	Suppression State
Error		The package reference 'Microsoft.Azure.Batch' should not specify a version.  Please specify the version in 'E:\development\microsoft\public\azure-sdk-for-net\eng\Packages.Data.props' or set VersionOverride to override the centrally defined version.	Microsoft.Azure.Batch.FileStaging	E:\development\microsoft\public\azure-sdk-for-net\sdk\batch\Microsoft.Azure.Batch.FileStaging\src\Microsoft.Azure.Batch.FileStaging.csproj	1	
```